relayer/chains/cosmos/event_parser_test.go

```diff
@@ -0,0 +1,315 @@
+package cosmos
+
+import (
+	"encoding/hex"
+	"strings"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	clienttypes "github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
+	conntypes "github.com/cosmos/ibc-go/v3/modules/core/03-connection/types"
+	chantypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	"github.com/cosmos/relayer/v2/relayer/processor"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+)
+
+func TestParsePacket(t *testing.T) {
+	testPacketTimeoutHeight := "1-1245"
+	testPacketTimeoutTimestamp := "1654033235600000000"
+	testPacketSequence := "1"
+	testPacketDataHex := "0123456789ABCDEF"
+	testPacketSrcChannel := "channel-0"
+	testPacketSrcPort := "port-0"
+	testPacketDstChannel := "channel-1"
+	testPacketDstPort := "port-1"
```

Comment:
Stylistically, I would rather see this as a `const` block to be more explicit about not allowing reassignment (and in the other new test functions too).

Reply:
done